src/tools/tidy/src/deps.rs

```diff
@@ -559,6 +561,21 @@ fn check_runtime_license_exceptions(
             if pkg.name == "fortanix-sgx-abi" && pkg.license.as_deref() == Some("MPL-2.0") {
                 continue;
             }
+
+            // This exception is due to the fact that the feature set of the
+            // `object` crate is different between rustc and libstd. In the
+            // standard library only a conservative set of features are enabled
+            // which notably does not include the `wasm` feature which pulls in
+            // this dependency. In the compiler, however, the `wasm` feature is
+            // enabled. This exception is intended to be here so long as the
+            // `EXCEPTIONS` below contains `wasmparser`, but once that goes away
```

Comment:
```suggestion
            // `EXCEPTIONS` above contains `wasmparser`, but once that goes away
```

Reply:
I don't see this change applied, yet it was marked as resolved.

Reply:
Oh dear sorry about that! I think I hit "resolve" and then forgot to actually apply the change assuming surely I'd remember. In any case I've actually applied it now this time.